### DISCUSSION TITLE
Border isn't showing

### DISCUSSION THREAD

**Discussion by imyanice** [ImageMagick/ImageMagick#5478] [Help]
Hi, first of all this is my template image I would like to the same thing in IM:
![​‍starting⁠‌​⁠‌​‌‍‌⁠​⁠‌⁠‍​‍⁠⁠‍‌​​‌​​​​​](https://user-images.githubusercontent.com/76485792/186104949-4a359ac7-9dd3-4423-91f7-fbb2a38514da.png)
 The Image must have:
- [x] A rounded corner of 12,
- [x] A shadow with 0.7 opacity, a size of 14, be centered,
- [ ] And a "glow" with the size 20, strength 1, with the RGB 200,200,255.

Here is my code so far:
```
magick input.png \
  \( +clone  -alpha extract \
    -draw 'fill black polygon 0,0 0,12 12,0 fill white circle 12,12 12,0' \
    \( +clone -flip \) -compose Multiply -composite \
    \( +clone -flop \) -compose Multiply -composite \
  \) -alpha off -compose CopyOpacity -composite \
  \( +clone -background black -shadow 0.7x14+0+0 \)  +swap -gravity center -compose over -composite output.png
```

So is there this "glow" in IM, or any equivalent ? I'm using IM latest on macOS Ventura
Thanks,

**Comment by snibgo** [comment]
> And a "glow" with the size 20, strength 1, with the RGB 200,200,255.

Please show an image with and without a "glow".

**Reply by imyanice** [reply]
![](https://imgs.search.brave.com/GcOnY0itlEl394hoU2Aii9LUWkbC0TiSUBFKZlDKTFg/rs:fit:500:619:1/g:ce/aHR0cHM6Ly9pLnBp/bmltZy5jb20vb3Jp/Z2luYWxzL2EwL2Vl/LzE3L2EwZWUxN2Qy/YmExMTY1NjVkMGJh/MmY5OTIwM2I2ZmE4/LmpwZw)

**Comment by snibgo** [comment]
I don't know what you mean by "glow". Perhaps you can explain it in words. Or you might show two versions of an image: one with no glow, and one with glow.

**Comment by fmw42** [comment]
Your shadow or glow does not show because you are using the wrong values. The intensity is in percent not a fraction. And you need go use the +X+Y distance values in addition the rolloff

-shadow IntensityxRolloff+Xdistance+Ydistance

see 

https://imagemagick.org/script/command-line-options.php#shadow
https://imagemagick.org/Usage/blur/#shadow

Try

![lena](https://user-images.githubusercontent.com/13458218/186208645-15bb072f-a2ff-4758-bf7a-5c1adf6250a6.png)


```
magick lena.png \
\( +clone  -alpha extract \
-draw 'fill black polygon 0,0 0,12 12,0 fill white circle 12,12 12,0' \
\( +clone -flip \) -compose Multiply -composite \
\( +clone -flop \) -compose Multiply -composite \
\) -alpha off -compose CopyOpacity -composite \
\( +clone -background black -shadow 70x14+14+14 \)  +swap -gravity center -compose over -composite output.png

```

![output](https://user-images.githubusercontent.com/13458218/186208675-57faae8e-2c50-4578-8325-e9bfef3bf4b0.png)


Is that what you want?


Or do you want a simple shadow as follows (use -layers merge not -composite)?

```
magick lena.png \
\( +clone  -alpha extract \
-draw 'fill black polygon 0,0 0,12 12,0 fill white circle 12,12 12,0' \
\( +clone -flip \) -compose Multiply -composite \
\( +clone -flop \) -compose Multiply -composite \
\) -alpha off -compose CopyOpacity -composite \
\( +clone -background black -shadow 70x14+14+14 \)  +swap -background none -compose over -layers merge output.png

```

![output](https://user-images.githubusercontent.com/13458218/186219152-d41cad67-7707-4ebf-8986-3a0f4d70facf.png)
